guide/sharding/additional-information.md

```diff
@@ -13,14 +13,31 @@ Here are some extra topics covered about sharding that might have raised concern
 
 ## Shard messages
 
-In order for shards to communicate, they must send messages to one another, as they are each their own process. You can listen for these messages by adding the following listener in your `index.js` file:
+<branch version="11.x">
+
+For shards to communicate, they have to send messages to one another, as they each have another process. You can listen for these messages by adding the following listener in your `index.js` file:
 
 ```js
 manager.on('message', (shard, message) => {
 	console.log(`Shard[${shard.id}] : ${message._eval} : ${message._result}`);
 });
 ```
 
+</branch>
+<branch version="12.x">
+
+For shards to communicate, they have to send messages to one another, as they each have another process. You can listen for these messages on the individual shards by adding the following lines in your `index.js` file:
+
+```js
+manager.shards.forEach(shard => {
+	shard.on('message', message => {
+		console.log(`Shard[${shard.id}] : ${message._eval} : ${message._result}`);
+	});
+});
+```
```

Comment:
This will be an empty collection on start, so we need to use the promise handled by `ShardingManager#spawn`, which resolves when the shards are finished spawning, like so (roughly): 

`manager.spawn().then(shards => shards.forEach(shard => shard.on('message', message => console.log(`Shard[${shard.id}] : ${message._eval} : ${message._result}`))));`